wear/src/main/java/io/homeassistant/companion/android/complications/ComplicationConfigActivity.kt

```diff
@@ -0,0 +1,58 @@
+package io.homeassistant.companion.android.complications
+
+import android.app.Activity
+import android.content.ComponentName
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.viewModels
+import androidx.wear.watchface.complications.datasource.ComplicationDataSourceService.Companion.EXTRA_CONFIG_COMPLICATION_ID
+import androidx.wear.watchface.complications.datasource.ComplicationDataSourceService.Companion.EXTRA_CONFIG_COMPLICATION_TYPE
+import androidx.wear.watchface.complications.datasource.ComplicationDataSourceService.Companion.EXTRA_CONFIG_DATA_SOURCE_COMPONENT
+import dagger.hilt.android.AndroidEntryPoint
+import io.homeassistant.companion.android.complications.views.LoadConfigView
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class ComplicationConfigActivity : ComponentActivity() {
+
+    @Inject
+    lateinit var presenter: ComplicationConfigPresenter
```

Comment:
This class shouldn't exist.  Everything should be in the view or the view model.